src/main/java/com/somena/vehicule/Vehicule.java

```diff
@@ -0,0 +1,60 @@
+package com.somena.vehicule;
+
+import com.somena.customer.Customer;
+
+public class Vehicule {
+
+    private Customer owner; // is owner long or Customer object?
```

Comment:
The owner is a customer object, so it should be fine to remove the comment

Reply:
The question here was should vehicule have access to the whole Customer object, or just the reference Id of the customer (which is a Long attribute)

Reply:
Reference only ideally, no one should have access to a customer object since there's a possibility of information leakage from an unwarranted source that has access to a vehicle object.

Reply:
modified it :+1: